form/create_custom_field_type.rst

```diff
@@ -4,23 +4,30 @@
 How to Create a Custom Form Field Type
 ======================================
 
-Symfony comes with a bunch of core field types available for building forms.
-However there are situations where you may want to create a custom form field
-type for a specific purpose. This article assumes you need a field definition
-that holds a shipping option, based on the existing choice field. This section
-explains how the field is defined and how you can customize its layout.
-
-Defining the Field Type
------------------------
-
-In order to create the custom field type, first you have to create the class
-representing the field. In this situation the class holding the field type
-will be called ``ShippingType`` and the file will be stored in the default location
-for form fields, which is ``App\Form\Type``.
-
-All field types must implement the :class:`Symfony\\Component\\Form\\FormTypeInterface`,
+Symfony comes with :doc:`tens of form types </reference/forms/types>` (called
```

Comment:
Yes ... this is because `:doc:`, `:class:`, etc. are "custom directives" which are not part of the standard RST format ... so that's why GitHub renders them wrongly. But they should be correct.